pkgs/applications/misc/klipperscreen/default.nix

```diff
@@ -1,28 +1,54 @@
-{ lib, stdenv, writeText, python3Packages, fetchFromGitHub, gtk3, gobject-introspection, gdk-pixbuf, wrapGAppsHook, librsvg }:
-python3Packages.buildPythonPackage rec {
+{ lib
+, python3
+, fetchFromGitHub
+, wrapGAppsHook
+, gobject-introspection
+, gitUpdater
+}: python3.pkgs.buildPythonApplication rec {
   pname = "KlipperScreen";
   version = "0.3.2";
+  format = "other";
 
   src = fetchFromGitHub {
     owner = "jordanruthe";
-    repo = pname;
+    repo = "KlipperScreen";
     rev = "v${version}";
     hash = "sha256-LweO5EVWr3OxziHrjtQDdWyUBCVUJ17afkw7RCZWgcg=";
   };
-  patches = [ ./fix-paths.diff ];
 
-  buildInputs = [ gtk3 librsvg ];
-  nativeBuildInputs = [ wrapGAppsHook gdk-pixbuf gobject-introspection ];
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook
+  ];
 
-  propagatedBuildInputs = with python3Packages; [ jinja2 netifaces requests websocket-client pycairo pygobject3 mpv six dbus-python numpy pycairo ];
+  pythonPath = with python3.pkgs; [
+    jinja2
+    netifaces
+    requests
+    websocket-client
+    pycairo
+    pygobject3
+    mpv
+    six
+    dbus-python
+  ];
```

Comment:
Any idea why this drops numpy/pycairo/librsvg?

Reply:
That's a list copied over from it's [requirements](https://github.com/jordanruthe/KlipperScreen/blob/master/scripts/KlipperScreen-requirements.txt)

numpy is not used in any way in the source, and not ever referenced — can you please elaborate it's inclusion?
pycairo is there
as for librsvg — it is propagated via wrapGApps

Reply:
Could've sworn I saw a distinctive failure to start without it...

Reply:
Please do! <3
`nixpkgs-review pr 232357` in a local nixpkgs copy will build and drop you into a shell with KlipperScreen

I've checked it with my local printer, and it does appear to be working, alas I didn't check bed mesh visualization yet, which miiiiight be using numpy

Reply:
If I had to guess where numpy is used, it'd be in bed mesh stuff. With that said, I found the commit which removed numpy and matplotlib: https://github.com/jordanruthe/KlipperScreen/commit/ce6158ad91ca1f556a20c3b429cc7ed241c69931

Seems they just re-implemented it without those deps at some point.

Reply:
Must've been leftovers from trying an older version, then.